src/App.vue

```diff
@@ -614,14 +614,16 @@ export default class App extends Mixins(
           }
         )
       }
+
+      console.log('*** got data')
+      this.haveData = true
     } catch (error) {
       // errors should be handled above
       console.error('Unhandled error in fetchData() =', error) // eslint-disable-line no-console
       // just fall through to finally()
     } finally {
-      this.haveData = true
```

Comment:
One theory I have is that there was an exception above and this finally block sets this flag, which is the gate to let the pages render. By moving this code to the bottom of the try block, it will only be run if everything in the try block succeeds (and will not be run if there was an exception).

NB: "finally" block runs after both try and catch blocks